packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -208,6 +231,10 @@ const Autocomplete = <
   const controlledStateRef = useRef(
     getControlState({ controlledValue: value, uncontrolledValue: defaultValue })
   );
+
+  const isVirtualized = useRef(
+    isVirtualizedProp !== undefined && isVirtualizedProp === true
+  );
```

Comment:
Can we write the boolean check to use the positive value first? Took me a bit to figure this out with the negative check first.

Also, do we need to write `=== true` when `&&` covers that? Since the type only allows booleans and `undefined`, we shouldn't have to account for `"x"` or `1` or something funky like that:
```suggestion
  const isVirtualized = useRef(
    isVirtualizedProp && isVirtualizedProp !== undefined
  );
```

Then again, maybe this doesn't show our intent. Instead of saying "this isn't undefined", should we look to make sure it's a boolean value?

```suggestion
  const isVirtualized = useRef(
    isVirtualizedProp && typeof isVirtualizedProp === "boolean"
  );
```

Reply:
Honestly, `Boolean(isVirtualizedProp)` probably works the best to account for the prop being an optional boolean. Changing it to that for simplicity's sake.